pom.xml

```diff
@@ -1895,6 +1895,17 @@
             <url>https://maven.restlet.com</url>
         </repository>
         -->
+        <repository>
+            <id>4science-nexus</id>
+            <url>https://nexus.4science.it/repository/maven-public</url>
```

Comment:
This should be removed now.  You should not need this now that you've switched to using `org.orcid:orcid-model` as the dependency.